docs/en/reference/caching.rst

```diff
@@ -42,7 +42,7 @@ object is closed:
 
     <?php
     $stmt = $conn->executeCacheQuery($query, $params, $types, new QueryCacheProfile(0, "some key"));
-    $data = $stmt->fetchAll();
+    $data = $stmt->fetchAllAssociative();
```

Comment:
Shouldn't the new docs be backported to the lower branch?

Reply:
Yes.